features/org.hibernate.search.eclipse.feature/feature.properties

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2010-2012 Red Hat, Inc. and others.
+# All rights reserved. This program and the accompanying materials 
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+# 
+# Contributors:
+#     bdshadow - Initial implementation.
+##############################################################################
+
+# "featureName" property - name of the feature
+featureName=Hibernate Search Tools
+
+# "providerName" property - name of the company that provides the feature
+providerName=bdshadow + JBoss by Red Hat
+
+# "description" property - description of the feature
+description=Hibernate Search Tools Eclipse Plugin provides index rebuild feature, \
+lucene documents overview and a tool to see the the output of analyzers . \
+
+copyright=JBoss, Home of Professional Open Source\n\
```

Comment:
@Sanne i was right, i've copied it from here: https://github.com/jbosstools/jbosstools-hibernate/blob/master/features/org.hibernate.eclipse.feature/feature.properties#L28. @koentsje @Sanne will fix it to the one, we have in jbosstools-openshift. It looks more valid